dirsrvtests/tests/suites/syncrepl_plugin/basic_test.py

```diff
@@ -301,11 +301,11 @@ def test_sync_repl_mep(topology, request):
 
     # checking that the cookie are in increasing and in an acceptable range (0..1000)
     assert len(cookies) > 0
-    prev = 0
+    prev = -1
```

Comment:
This looks like it shouldn't be in this PR

Reply:
I messed when trying to rebase to the master and 3 unwanted commits were pushed in the original pull request.
I rebased to remove them this morning so the extra files are gone. 
I will wait until the work is nearly done before re merging with the master (as the merge may be painful)